nmea.h

```diff
@@ -169,6 +170,8 @@ class NmeaFormat : public Format
 
   int wpt_not_added_yet{};
 
+  std::map<std::string, int> counter;
```

Comment:
For consistency I'd prefer to stick with Qt containers unless there is a compelling advantage to std lib in a particular use case.

Reply:
Agreed. Changed and renamed.